hibernate-envers/hibernate-envers.gradle

```diff
@@ -26,10 +26,6 @@ dependencies {
 
     testImplementation project( ':hibernate-testing' )
 
-    // todo : to get rid of these
-    testImplementation testLibs.junit4
-    testImplementation testLibs.junit4Engine
-
     testImplementation project(':hibernate-testing')
```

Comment:
Can you also please remove the duplicate `testImplementation project(':hibernate-testing')` please? I must've added it by mistake.